content/commands/ft.search/index.md

```diff
@@ -312,7 +316,11 @@ syntax_str: "query [NOCONTENT] [VERBATIM] [NOSTOPWORDS] [WITHSCORES] [WITHPAYLOA
 title: FT.SEARCH
 ---
 
-Search the index with a textual query, returning either documents or just ids
+Search the index with a textual query, returning either documents or just ids.
+
+{{< note >}}
+This command will only return keys that the user has access to.
```

Comment:
What type of access? Read?
@raz-mon

Reply:
@LiorKogan Yes

Reply:
@dwdougherty please change to "has a read access to"

Reply:
Change "only return" to "return only"?